### PR TITLE
[feature-6] Crete second card

### PR DESCRIPTION
**Pull request by iamandiradu**
This PR adds the List component.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,509 @@
+{
```
Comment (andreifilip123) [inline .eslintrc]: This file shouldn't be committed. Maybe you can add it to .gitignore?

File: src/core/list/components/Title.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Box, Typography } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+
+const primary = {
+  fontSize: 36,
+  fontWeight: '700',
+  lineHeight: 1.11,
+  letterSpacing: '-0.03em',
```
Comment (andreifilip123) [inline src/core/list/components/Title.jsx]: Usually, letter spacings are expressed in PX. (You wouldn't want your letter spacing to be relative to the screen size)

File: src/core/list/components/Title.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
```
Comment (andreifilip123) [inline src/core/list/components/Title.jsx]: Components should be named `lower-case-component-with-kebab-case.jsx`

File: src/core/list/components/Title.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import { Box, Typography } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
```
Comment (andreifilip123) [inline src/core/list/components/Title.jsx]: you can import makeStyles from `@material-ui/core`

File: src/core/list/components/Title.jsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+const primary = {
+  fontSize: 36,
+  fontWeight: '700',
+  lineHeight: 1.11,
+  letterSpacing: '-0.03em',
+  textAlign: 'left',
```
Comment (andreifilip123) [inline src/core/list/components/Title.jsx]: Isn't left the default ? I don't think you need to specifically mention it unless you overwrite something

File: src/core/list/components/Title.jsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  marginBottom: 10,
+}
+const secondary = {
+  fontSize: 16,
+  fontWeight: '400',
+  lineHeight: 1.625,
+  letterSpacing: '0em',
```
Comment (andreifilip123) [inline src/core/list/components/Title.jsx]: After a zero length, the unit identifier is optional
https://www.w3.org/TR/CSS22/syndata.html#length-units

**Review by andreifilip123 — CHANGES_REQUESTED** [review]

File: .eslintrc (hunk 1 above)
Comment (radubuciuceanu) [reply .eslintrc]: Done.

File: src/core/list/components/Title.jsx (hunk 3 above)
Comment (radubuciuceanu) [reply src/core/list/components/Title.jsx]: Done.

File: src/core/list/components/Title.jsx (hunk 4 above)
Comment (radubuciuceanu) [reply src/core/list/components/Title.jsx]: Done.

File: src/core/list/components/Title.jsx (hunk 5 above)
Comment (radubuciuceanu) [reply src/core/list/components/Title.jsx]: Done.

File: src/core/list/components/Title.jsx (hunk 6 above)
Comment (radubuciuceanu) [reply src/core/list/components/Title.jsx]: Done.

File: src/core/list/components/Title.jsx (hunk 2 above)
Comment (radubuciuceanu) [reply src/core/list/components/Title.jsx]: Will handle this one in further issues.

Comment (andreifilip123) [reply src/core/list/components/Title.jsx]: Can't we just do `letterSpacing: -1` now ?

**Review by radubuciuceanu — APPROVED** [review]

**Comment by iamandiradu** [comment]
Sorry guys. Didn't have time to correct this. Thanks for your work!

**Pull request merged**